experimental/examples/attributable-map-demo/src/fluid-object/index.ts

```diff
@@ -0,0 +1,47 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { DataObject, DataObjectFactory } from "@fluidframework/aqueduct";
+import { IFluidHandle } from "@fluidframework/core-interfaces";
+import { AttributableMap } from "@fluid-experimental/attributable-map";
+
+export class AttributableMapPage extends DataObject {
```

Comment:
Consider naming something like `TimestampWatcher` to indicate the encapsulated functionality.